tests/stress_tests/scripts/run_memcheck.py

```diff
@@ -148,6 +148,7 @@ def main():
     returncode, _ = run([sys.executable, args.gtest_parallel] +
                         (['--output_dir', f'{args.output_dir}'] if args.output_dir else []) +
                         (['--workers', f'{args.workers}'] if args.workers else []) +
+                        ['--timeout', str(4 * 60 * 60)] +
```

Comment:
timeout limit should not be hardcoded in this script. User should be able to call  run_memcheck and specify timeout.

Reply:
Done